README.md

```diff
@@ -1,11 +1,82 @@
+![Aragon](https://res.cloudinary.com/dacofvu8m/image/upload/v1677353961/Aragon%20CodeArena/osx_blue_logo_lqrvkr.png)
+
 # Aragon JS SDK
 
-This folder contains all the JS packages available on NPM:
+Welcome to the Aragon JS SDK! This SDK allows you to interact with Aragon OSx
+using JavaScript. Whether you are a developer wanting to build on top of Aragon,
+or looking to contribute to the SDK itself, you are in the right place! This
+folder contains all the JS packages available on NPM:
 
 - [Client](./modules/client)
 - [IPFS](./modules/ipfs)
 - [Common](./modules/common)
 
+## Developer Quick Start
+
+The Aragon OSx SDK is Javascript agnostic, which means you can use it with any
+Javascript framework, including popular ones like React, Vite, or Vue. The
+examples in this readme are written using NodeJS
+
+First things first install the client with
```

Comment:
"First, let's install the SDK using this command in our terminal:"

Reply:
Would be great if we could go through the entire flow of creating a project using the SDK so there's minimal confusion for devs. Also, working with a numbered format helps visualize the steps easier. 

For example, having a project to install the package in is a pre-requisite.

I'd suggest adding something like:

1) Specify the repo you'd like to install the SDK in
You can install the Aragon OSx SDK in any project which contains a `package.json` or a `yarn.lock`. If you don't have a project already created, you can initialize one calling on the following command in your terminal:
```bash
npm init 
```
That will generate a `package.json` file for you to start running your Javascript project.

2. Install the Aragon OSx SDK 
To install the Aragon OSx SDK into your project, you should run the following command in your terminal:
```bash
npm install @aragon/sdk-client
```
or 
```bash
yarn add @aragon/sdk-client
```

3. Config for the SDK
.....

4. Using the SDK
....